src/Graphics/src/Graphics/Platforms/Android/PlatformImage.cs

```diff
@@ -156,7 +156,24 @@ public IImage ToPlatformImage()
 
 		public static IImage FromStream(Stream stream, ImageFormat formatHint = ImageFormat.Png)
 		{
-			var bitmap = BitmapFactory.DecodeStream(stream);
+			Bitmap bitmap;
+
+			if (stream is null)
+			{
+				return null;
+			}
+
+			using (var memoryStream = new MemoryStream())
```

Comment:
Here you're allocating a memory stream just to convert it into a byte array later, why not just read the Stream directly into a `byte []`?

Reply:
Yeah, I see, but you don't need to create a memory stream in order to it. You can read directly from the `Stream`, something like


```cs
```